NBitcoin/Network.cs

```diff
@@ -76,7 +76,8 @@ public enum Base58Type
 	public enum Bech32Type
 	{
 		WITNESS_PUBKEY_ADDRESS,
-		WITNESS_SCRIPT_ADDRESS
+		WITNESS_SCRIPT_ADDRESS,
+		BLINDED_ADDRESS

```

Comment:
no shitcoin code in NBitcoin

Reply:
I put this one here because there is already `Base58Type.BLINDED_ADDRESS`

Reply:
fuck ok